rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -165,6 +165,28 @@ object ProcNormalizeMatcher {
       case p: PAnd => binaryExp(p.proc_1, p.proc_2, input, EAnd)
       case p: POr => binaryExp(p.proc_1, p.proc_2, input, EOr)
 
+      case p: PSend => {
+        import scala.collection.JavaConverters._
+        val nameMatchResult = NameNormalizeMatcher.normalizeMatch(
+          p.name_,
+          NameVisitInputs(input.env, input.knownFree))
+        val initAcc = (List[Par](), ProcVisitInputs(Par(), input.env, nameMatchResult.knownFree))
+        val dataResults = (initAcc /: p.listproc_.asScala.toList.reverse)(
+          (acc, e) => {
+            val procMatchResult = normalizeMatch(e, acc._2)
+            (procMatchResult.par :: acc._1,
+             ProcVisitInputs(Par(), input.env, procMatchResult.knownFree))
+          }
+        )
+        val persistent = p.send_ match {
+          case _: SendSingle => false
+          case _: SendMultiple => true
```

Comment:
My immediate reaction to seeing this was "oh so you can either send a single value or a tuple - as in x!(5) vs x!(5,5)" but then I realized this meant x!(5) vs x!!(5). Maybe a rename? But eh maybe we shouldn't touch the grammar too frequently.